app/assets/stylesheets/app/profile.scss

```diff
@@ -0,0 +1,150 @@
+$team-link-color: #5e9aff;
+$dark-text-color: #49465e;
+$darker-text-color: #5b596e;
+$border-color: #e7e4ed;
+
+.profile {
+  &__section {
+    &+.profile__section {
+      margin-top: 5%;
+    }
+  }
+
+  &__header {
+    flex: 1;
+    padding: 0 2%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+  }
+
+  &__picture {
+    flex: 2;
```

Comment:
Properties should be ordered align-items, display, flex, flex-direction, padding